src/PowerShellEditorServices/Services/PowerShellContext/Handlers/GetVersionHandler.cs

```diff
@@ -91,7 +91,7 @@ private async Task CheckPackageManagement()
 
                 if (_powerShellContextService.CurrentRunspace.Runspace.SessionStateProxy.LanguageMode != PSLanguageMode.FullLanguage)
                 {
-                    _languageServer.Window.ShowWarning("You have an older version of PackageManagement known to cause issues with the PowerShell extension. Please run the following command in a new Windows PowerShell session and then restart the PowerShell extension: `Install-Module PackageManagement -Force -AllowClobber -MinimumVersion 1.4.6`");
+                    _languageServer.Window.LogWarning("You have an older version of PackageManagement known to cause issues with the PowerShell extension. Please run the following command in a new Windows PowerShell session and then restart the PowerShell extension: `Install-Module PackageManagement -Force -AllowClobber -MinimumVersion 1.4.6`");
```

Comment:
This doesn't seem right... the spec says there's a `ShowMessage` and a `LogMessage`...

https://microsoft.github.io/language-server-protocol/specifications/specification-current/#window_showMessage

@david-driscoll did this API change?

Reply:
hmm... not intentionally.

Reply:
taking a look at the assembly.

Reply:
yep, I deleted one to many methods!

Reply:
I will bring those back and get 0.17.4 out to fix that.

Reply:
as a work around for the moment you can use ShowMessage and give it the correct message type.

Reply:
New package is out, this should restore that function.
https://www.nuget.org/packages/OmniSharp.Extensions.LanguageServer/0.17.4